apps/backend/src/factory/EssDownloadService.ts

```diff
@@ -0,0 +1,107 @@
+import querystring from 'querystring';
+import { Readable } from 'stream';
+import { ReadableStream } from 'stream/web';
+
+import { logger } from '@user-office-software/duo-logger';
+import contentDisposition from 'content-disposition';
+import { Request, Response, NextFunction } from 'express';
+
+import { Role } from '../models/Role';
+import {
+  MetaBase,
+  DownloadType,
+  PDFType,
+  XLSXType,
+  ZIPType,
+  DownloadService,
+} from './DownloadService';
+
+const ENDPOINT = process.env.USER_OFFICE_FACTORY_ENDPOINT;
+
+export class EssDownloadService implements DownloadService {
```

Comment:
I think we can call it DefaultDownloadService instead of EssDownloadService, as it is not an ESS-specific service.